bundle/manifests/ibm-secretshare-operator.clusterserviceversion.yaml

```
@@ -173,4 +173,7 @@ spec:
   maturity: alpha
   provider:
     name: IBM
+  relatedImages:
+  - image: quay.io/opencloudio/ibm-secretshare-operator:1.7.0
+    name: ibm-secretshare-operator
```

The name should follow CICD convention right?

```suggestion
    name: IBM_SECRETSHARE_OPERATOR_IMAGE
```

---

Nice catch, updated, Thank you!